src/sentry/static/sentry/app/index.js

```diff
@@ -1,15 +1,47 @@
 import '@babel/polyfill';
 
-// Initial data hydration. The __initialData will be set by the django backend
-// serving sentry.
-if (window.__initialData) {
-  const {distPrefix, csrfCookieName, sentryConfig, userIdentity} = window.__initialData;
+const BOOTSTRAP_URL = '/api/client-config/';
 
+const bootApplication = data => {
+  const {distPrefix, csrfCookieName, sentryConfig, userIdentity} = data;
+
+  // TODO(epurkhiser): Would be great if we could remove some of these from
+  // existing on the window object and instead pass into a bootstrap function.
+  // We can't currently do this due to some of these globals needing to be
+  // available for modules imported by the bootstrap.
```

Comment:
If bootstrap exported a function to window could we call that function with this state? That would let us remove more globals.

Reply:
Right, that's what we should do, but:

> We can't currently do this due to some of these globals needing to be available for modules imported by the bootstrap.